lib/ansible/modules/system/aix_filesystem.py

```diff
@@ -0,0 +1,600 @@
+        if not module.check_mode:
+            rc, crfs_out, err = module.run_command(
+                "%s -v %s -m %s %s %s %s %s %s -p %s %s -a %s" % (
+                    crfs_cmd, fs_type, filesystem, vg, device, mount_group,
+                    auto_mount, account_subsystem, permissions, size,
+                    attributes))
+
+            if rc == 10:
+                module.exit_json(
+                    msg="Using a existent previously defined logical volume, "
+                        "volume group needs to be empty. %s" % err)
+
+            elif rc != 0:
+                module.fail_json(msg="Failed to run crfs.", rc=rc, err=err)
+
+            else:
+                changed = True
+                return changed, crfs_out
+        else:
+            changed = True
+            msg = ''
+
+            return changed, msg
+
+
+def remove_fs(module, filesystem, rm_mount_point):
+    """ Remove an LVM file system or NFS entry. """
+
+    # Command parameters.
+    rm_mount_point_opt = {
+        True: '-r',
+        False: ''
+    }
+
+    rm_mount_point = rm_mount_point_opt[rm_mount_point]
+
+    rmfs_cmd = module.get_bin_path('rmfs', True)
+    if not module.check_mode:
+        rc, rmfs_out, err = module.run_command(
+            "%s -r %s %s" % (rmfs_cmd, rm_mount_point, filesystem))
+        if rc != 0:
+            module.fail_json(msg="Failed to run rmfs.", rc=rc, err=err)
+        else:
+            changed = True
+            msg = rmfs_out
+            if not rmfs_out:
+                msg = "File system %s removed." % filesystem
+
+            return changed, msg
+    else:
+        changed = True
+        msg = ''
+
+        return changed, msg
+
+
+def mount_fs(module, filesystem):
+    """ Mount a file system. """
+    mount_cmd = module.get_bin_path('mount', True)
+
+    if not module.check_mode:
+        rc, mount_out, err = module.run_command(
+            "%s %s" % (mount_cmd, filesystem))
+        if rc != 0:
+            module.fail_json("Failed to run mount.", rc=rc, err=err)
+        else:
+            changed = True
+            msg = "File system %s mounted." % filesystem
+
+            return changed, msg
+    else:
+        changed = True
+        msg = ''
+
+        return changed, msg
+
+
+def unmount_fs(module, filesystem):
+    """ Unmount a file system."""
+    unmount_cmd = module.get_bin_path('unmount', True)
+
+    if not module.check_mode:
+        rc, unmount_out, err = module.run_command(
+            "%s %s" % (unmount_cmd, filesystem))
+        if rc != 0:
+            module.fail_json("Failed to run unmount.", rc=rc, err=err)
+        else:
+            changed = True
+            msg = "File system %s unmounted." % filesystem
+
+            return changed, msg
+    else:
+        changed = True
+        msg = ''
+
+        return changed, msg
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            account_subsystem=dict(type='bool', default=False),
```

Comment:
Please test if `default=False` is the default value when using type `bool`.

Reply:
No, the default is None. So you can tell if the parameter was set or not.